websites/N/Nebula/presence.ts

```diff
@@ -0,0 +1,285 @@
+const presence = new Presence({
+		clientId: "1212664221788274698",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://i.imgur.com/VnwSEyu.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = document.location,
+		[showButtons] = await Promise.all([
+			presence.getSetting<boolean>("buttons"),
+		]),
```

Comment:
While getting one promise, don't use Promise.all
```suggestion
		showButtons = 
			await presence.getSetting<boolean>("buttons"),
```